docs/contributors.html

```diff
@@ -1 +1,235 @@
+      "repos_url": "https://api.github.com/users/digilou/repos",
+      "events_url": "https://api.github.com/users/digilou/events{/privacy}",
+      "received_events_url": "https://api.github.com/users/digilou/received_events",
+      "type": "User",
+      "site_admin": false,
+      "contributions": 7
+    },
+    {
+      "login": "amr08",
+      "id": 17239196,
+      "avatar_url": "https://avatars3.githubusercontent.com/u/17239196?v=4",
+      "gravatar_id": "",
+      "url": "https://api.github.com/users/amr08",
+      "html_url": "https://github.com/amr08",
+      "followers_url": "https://api.github.com/users/amr08/followers",
+      "following_url": "https://api.github.com/users/amr08/following{/other_user}",
+      "gists_url": "https://api.github.com/users/amr08/gists{/gist_id}",
+      "starred_url": "https://api.github.com/users/amr08/starred{/owner}{/repo}",
+      "subscriptions_url": "https://api.github.com/users/amr08/subscriptions",
+      "organizations_url": "https://api.github.com/users/amr08/orgs",
+      "repos_url": "https://api.github.com/users/amr08/repos",
+      "events_url": "https://api.github.com/users/amr08/events{/privacy}",
+      "received_events_url": "https://api.github.com/users/amr08/received_events",
+      "type": "User",
+      "site_admin": false,
+      "contributions": 3
+    },
+    {
+      "login": "desdemonhu",
+      "id": 26439524,
+      "avatar_url": "https://avatars0.githubusercontent.com/u/26439524?v=4",
+      "gravatar_id": "",
+      "url": "https://api.github.com/users/desdemonhu",
+      "html_url": "https://github.com/desdemonhu",
+      "followers_url": "https://api.github.com/users/desdemonhu/followers",
+      "following_url": "https://api.github.com/users/desdemonhu/following{/other_user}",
+      "gists_url": "https://api.github.com/users/desdemonhu/gists{/gist_id}",
+      "starred_url": "https://api.github.com/users/desdemonhu/starred{/owner}{/repo}",
+      "subscriptions_url": "https://api.github.com/users/desdemonhu/subscriptions",
+      "organizations_url": "https://api.github.com/users/desdemonhu/orgs",
+      "repos_url": "https://api.github.com/users/desdemonhu/repos",
+      "events_url": "https://api.github.com/users/desdemonhu/events{/privacy}",
+      "received_events_url": "https://api.github.com/users/desdemonhu/received_events",
+      "type": "User",
+      "site_admin": false,
+      "contributions": 3
+    }
+  ];
+
+
+
+      
+      
+var html = '<table class="table table-bordered"><thead><tr><th>Name</th><th>GitHub</th><tbody>'
+for (var i in json_data){
+    html = html+"<tr><td><img style='width:150px;' src='" +json_data[i]["avatar_url"]+"'>"+"</td><td><a href='"+json_data[i]["html_url"]+"'>"+json_data[i]["login"]+"</a></td></tr>"
+}
+html=html+'</tbody></table>'
+document.getElementById("display").innerHTML= html;
+
+</script>
+
+<style>
+.jumbotron{
+ background-color: #4885ed;   
+ color: white;
+}
+</style>
```

Comment:
@tanyagupta Thanks for doing this! Can you migrate the scripts and styles out of this file?

Reply:
Thanks @digilou. Not ready for review yet :) so has some interim coding. will clean up and tag when done.